wv2util/MainWindow.xaml.cs

```diff
@@ -336,5 +355,90 @@ private void WatchForChangesTimer_Elapsed(object sender, ElapsedEventArgs e)
             }
             m_previousHostAppEntries = currentHostAppEntries;            
         }
+
+        private string newsLink = "https://api.github.com/repos/MicrosoftEdge/WebView2Announcements/issues";
+        private async Task<string> GetNewsData()

```

Comment:
Please add 'Async' suffix to async method names.
```suggestion
        private async Task<string> GetNewsDataAsync()
```